[2023-10-25 11:35:31] System.Exception: test
StackTrace:    at Datadrasil.Menu.<>c.<Main>b__0_0() in C:\Users\ludvig.johansson1\Main\memodrasil\Datadrasil\Datadrasil\MenuSystem\Menu.cs:line 19
   at Datadrasil.MenuItem.Execute() in C:\Users\ludvig.johansson1\Main\memodrasil\Datadrasil\Datadrasil\MenuSystem\MenuItem.cs:line 26
   at Datadrasil.MenuManager.Execute() in C:\Users\ludvig.johansson1\Main\memodrasil\Datadrasil\Datadrasil\MenuSystem\MenuManager.cs:line 32
   at Datadrasil.MenuItem.Execute() in C:\Users\ludvig.johansson1\Main\memodrasil\Datadrasil\Datadrasil\MenuSystem\MenuItem.cs:line 31
   at Datadrasil.MenuManager.Execute() in C:\Users\ludvig.johansson1\Main\memodrasil\Datadrasil\Datadrasil\MenuSystem\MenuManager.cs:line 32
   at Datadrasil.MenuItem.Execute() in C:\Users\ludvig.johansson1\Main\memodrasil\Datadrasil\Datadrasil\MenuSystem\MenuItem.cs:line 31
   at Datadrasil.MenuManager.Execute() in C:\Users\ludvig.johansson1\Main\memodrasil\Datadrasil\Datadrasil\MenuSystem\MenuManager.cs:line 32
   at Datadrasil.Program.Main() in C:\Users\ludvig.johansson1\Main\memodrasil\Datadrasil\Datadrasil\Program.cs:line 19
------------------------------
